<launch>

	<include file="$(find pointcloud_utils)/launch/cloud_stamper.launch" />
	
    <node pkg="pointcloud_utils" type="point_cloud_point_saver.py" name="pc_point_saver"
          required="true" output="screen" cwd="node">
          <param name="target_frame" value="base" />
          <param name="file_name"    value="innoviz_points" />
          <param name="folder_name"  value="points_test" />
          <param name="file_type" 	 value="xyz" />
		  <param name="cloud_topic" 		value="/leader_points/truth/innoviz" />
		  <param name="file_list_name" value="file_list.txt" />
    </node>
</launch> 